### PR TITLE
[T7][T17-C3]

### PR DESCRIPTION
**Pull request by nyannnnnnn**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -286,20 +286,32 @@ Use case ends.
...
-> 3a1. AddressBook shows an error message <br>
+> 3a1. Task Manager shows an error message <br>
   Use case resumes at step 2
 
-{More to be added}
+#### Use case: Search task
+
+**MSS**
+
+1. User type the correct command in the command box(search KEYWORD)
```
Comment (pixelducky) [inline docs/DeveloperGuide.md]: There is no need to be too specific about the syntax of the search command. It is assumed to be correct. Any invalid cases will be stated in the extensions. Perhaps you could rephrase this to follow the writing style of the previous use cases above. i.e. `1. User requests to list persons`

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -286,20 +286,32 @@ Use case ends.
...
+2. Task Manager shows a task or more tasks which match the KEYWORD.<br>
+Use case ends.
+
+**Extensions**
+
+1a. The user don't type the correct keyword
+
```
Comment (pixelducky) [inline docs/DeveloperGuide.md]: so what happens after 1a. ?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -286,20 +286,32 @@ Use case ends.
...
+ 
+ 
 
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java `1.8.0_60` or higher installed.
-2. Should be able to hold up to 1000 persons.
+2. Should be able to hold up to 1000 tasks.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
```
Comment (pixelducky) [inline docs/DeveloperGuide.md]: this shouldn't be here anymore if you guys are working towards making your command input to be more natural

File: docs/AboutUs.md [hunk 4]
```diff
@@ -1,52 +1,42 @@
 # About Us
```
Comment (pixelducky) [inline docs/AboutUs.md]: good job in updating this 👍

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -256,26 +256,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+`* * *` | user | add a new task |
+`* * *` | user | delete a task | remove task that I have done
+`* * *` | user | find a task by name | locate details of tasks without having to go through the entire list
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
```
Comment (pixelducky) [inline docs/DeveloperGuide.md]: your todo app can hide contact details? :P

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -312,5 +324,10 @@ Use case ends.
...
+   | Todoist| Any.Do | Wunderlist
+-------- | :-------- | :--------- | :-----------
+`Platforms` | All platforms | iOS, Android, Google Chrome | iPhone, iPad, Android, Windows, Kindle, Web
+`Price` | Free / Premium ($28.99/year) | Free / Premium ($45/year or $5 per month) | Free / Pro ($59.88/year)
+`Pros` | Good user interface<br>Fastest task manager app Works offline<br> Can set priorities by choosing from 1 to 4. <br>Can drag and drop tasks <br>Can create a “Project” with emojis in its name  Supports text formatting| Minimalistic design Can “Star” a task to mark it as high priority Runs at high speed Has in-built speech recognition <br>Can drag and drop tasks <br>Can swipe a task to the right to mark it as complete<br> Can swipe a completed task to the left to add it again to the list. <br>Can attach files from your Dropbox and Google Drive  | Simple-to-use <br>Can “Star” a task to set a priority to a task Hashtags feature<br> Can add comments and attachments Supports email reminder Powerful search function Can restore deleted lists. <br>Can manually create backups and import data from your backups.<br> Can attach files from your Dropbox.
```
Comment (pixelducky) [inline docs/DeveloperGuide.md]: Make sure every new point starts on a new line. Even better if you can bullet point it.

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -312,5 +324,10 @@ Use case ends.
...
+`Cons` | The key features are all part of premium plan <br>Not compatible with Windows Phone Search function is limited in free plan <br>Doesn’t have a backup option for free users | Not intuitive to use Doesn’t hide a task when it’s marked as complete.<br> Doesn’t support email reminders Doesn’t have a Windows app.<br> Doesn’t have a backup option. Doesn’t support emojis in your list names. Doesn’t support markdown.| Slow to load <br>Doesn’t show last synced time <br>Doesn’t support text formatting
```
Comment (pixelducky) [inline docs/DeveloperGuide.md]: Make sure every new point starts on a new line. Even better if you can bullet point it.

**Comment by pixelducky** [comment]
@nyannnnnnn @Hanxnow77 @ghurabah93 @evilmtv Please close PR after reading comments. Thanks.

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -286,20 +286,32 @@ Use case ends.
 
 3a. The given index is invalid
 
-> 3a1. AddressBook shows an error message <br>
+> 3a1. Task Manager shows an error message <br>
```
Comment (ghurabah93) [inline docs/DeveloperGuide.md]: show error message first then return to step 2

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (ghurabah93) [inline docs/DeveloperGuide.md]: Sorry, we can't do that as we want to show a nice table :)

File: docs/DeveloperGuide.md (hunk 8 above)
Comment (pixelducky) [inline docs/DeveloperGuide.md]: I see. Make sure this is mentioned clearly in your latest DevGuide!

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (pixelducky) [inline docs/DeveloperGuide.md]: Okay then at least make sure that different points start off on a new line

**Comment by pixelducky** [comment]
@nyannnnnnn @ghurabah93 @Hanxnow77 @evilmtv I noticed that the repo issue tracker is still not enabled. Switch on the Issue Tracker via the repo's Settings page
